src/stores/security-google-recaptcha.md

```diff
@@ -2,53 +2,47 @@
+  - _reCAPTCHA v3 Invisible_ — Uses an algorithm to rate user interactions and determines the liklihood that the user is human based on a score.
 
-If you have the standard [Magento CAPTCHA]({% link stores/security-captcha.md %}) enabled, reCAPTCHA will not override it. You can have both enabled on your Magento instance without issues.
+   - _reCAPTCHA v2 Invisible_ — (**recommended**) Performs background verification without user interaction. Users and customers are automatically verified, but might be required to select specific images to complete a challenge.
 
-![Google reCAPTCHA - styles]({% link images/images/recaptcha-styles.png %}){: .zoom}
-_Google reCAPTCHA v2 Styles_
+   - _reCAPTCHA v2 ("I am not a robot")_ — Validates requests with the _"I'm not a robot"_ checkbox.
 
 {:.bs-callout-warning}
-Before Google reCAPTCHA can be configured, you must ensure that your `PHP.ini` file includes the following setting: `allow_url_fopen = 1`. This may require developer assistance. See [Required PHP Settings](https://devdocs.magento.com/guides/v2.3/install-gde/prereq/php-settings.html){:target="_blank"}.
+Before Google reCAPTCHA can be configured, ensure that your `PHP.ini` file includes the following setting: `allow_url_fopen = 1`. This may require developer assistance. See [Required PHP Settings](https://devdocs.magento.com/guides/v2.3/install-gde/prereq/php-settings.html){:target="_blank"}.
 
 ## Step 1: Generate Google reCAPTCHA keys
 
-Google reCAPTCHA requires a pair of API keys to enable. You can get these keys free of charge through the [reCAPTCHA site][2]. Before generating the keys, consider the type of reCAPTCHA you want to use.
+Google reCAPTCHA requires a pair of API keys to enable. You can get these keys free of charge through the [reCAPTCHA site][2]. Before generating the keys, you must know the type of reCAPTCHA that you want to use.
 
 1. Open the [Google reCAPTCHA page][2] and log in to your account.
 
-1. Create/Register a new site.
+1. Enter a **Label** to identify the keys for internal reference.
 
-1. Enter a **Label** for the keys.
+   You need one set of keys for each reCAPTCHA type that is used in your Magento installation. For example: `Magento v2 Invisible`
 
-   For example, enter `Magento v2 Invisible`. You only need one set of keys per each reCAPTCHA type for your entire Magento instance.
+1. Choose the **reCAPTCHA type** that you want to use.
 
-1. Choose the type of Google reCAPTCHA that you want to use:
+   - _reCAPTCHA v3 Invisible_
+   - _reCAPTCHA v2 Invisible_
+   - _reCAPTCHA v2 ("I am not a robot")_
 
-   - `reCAPTCHA v3` — Verify requests with a score
-   - `reCAPTCHA v2` — Verifies with a button click or image selections
+1. Enter your store's **Domain**. For example: mystore.com
```

Comment:
```suggestion
1. For **Domain**, enter your store's domain. For example: mystore.com
```

Reply:
I think it's better to incorporate the field name in the instruction, if it can be done naturally, as in this case. But I'll go ahead and do it for consistency at this point.